automation/deploy_hco.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HCO_VERSION=1.4.0
+CMD=oc
+NS=kubevirt-hyperconverged
+
+# Create a catalog source
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-unstable-catalog-source
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: "quay.io/kubevirt/hyperconverged-cluster-index:${HCO_VERSION}"
+  displayName: Kubevirt Hyperconverged Cluster Operator
+  publisher: Kubevirt Project
+EOF
+
+# Create the kubevirt-hyperconverged namespace
+cat <<EOF | ${CMD} apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: "${NS}"
```

Comment:
same for namespace name. anyway space is not allowed in kubernetes object name.

Reply:
same - why do we care?